src/main/cms/cms_menu_failsafe.c

```diff
@@ -0,0 +1,90 @@
+/*
+ * This file is part of Cleanflight and Betaflight.
+ *
+ * Cleanflight and Betaflight are free software. You can redistribute
+ * this software and/or modify this software under the terms of the
+ * GNU General Public License as published by the Free Software
+ * Foundation, either version 3 of the License, or (at your option)
+ * any later version.
+ *
+ * Cleanflight and Betaflight are distributed in the hope that they
+ * will be useful, but WITHOUT ANY WARRANTY; without even the implied
+ * warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <stdbool.h>
+#include <stdint.h>
+#include <string.h>
+#include <ctype.h>
+
+#include "platform.h"
+
+#if (defined(USE_CMS) && defined(USE_CMS_FAILSAFE_MENU))
+
+#include "cms/cms.h"
+#include "cms/cms_types.h"
+#include "cms/cms_menu_failsafe.h"
+
+#include "config/feature.h"
+
+#include "fc/config.h"
+
+#include "flight/failsafe.h"
+
+#include "rx/rx.h"
+
+uint8_t failsafeConfig_failsafe_procedure;
+uint8_t failsafeConfig_failsafe_delay;
+uint8_t failsafeConfig_failsafe_off_delay;
+uint16_t failsafeConfig_failsafe_throttle;
+
+static long cmsx_Failsafe_onEnter(void)
+{
+    failsafeConfig_failsafe_procedure = failsafeConfig()->failsafe_procedure;
+    failsafeConfig_failsafe_delay = failsafeConfig()->failsafe_delay;
+    failsafeConfig_failsafe_off_delay = failsafeConfig()->failsafe_off_delay;
+    failsafeConfig_failsafe_throttle = failsafeConfig()->failsafe_throttle;
```

Comment:
I like to align the assignments by the equal symbol, but this only a preference of mine. I align too the variables declaration.

Reply:
We generally avoid extra whitespace in statements because it can lead to merging problems. But aligning array declarations like above is commonly used.

Reply:
I honestly do not like tabular alignment, and I am thinking of proposing a rule stipulating it isn't used. The most important reason for this is that, if tabular alignment is used, addition of a longer element or removal of the longest element will require potentially 100s of other lines to be changed, creating a lot of unneeded change and making reviews harder to do.

Reply:
True. Aesthetically is better, but I understand that in collaborative projects changes like the one you say complicates some things... I think it's a good idea to add the rule.